bin/tictactoe

```diff
@@ -1,3 +1,65 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+
+
+def play_game
+  opening_statments
+end
+
+def opening_statments
+  puts "Welcome to Tic Tac Toe!"
+  puts "Choose between 1-3"
+  puts "1. Watch CPU play"
+  puts "2. Play vs CPU"
+  puts "3. Play with friend"
+
+  input = ""
+  while input != "exit"
+    input = gets.chomp
+
+    if input == "1"
+      cpu
+    elsif input == "2"
+      one_player
+    elsif input == "3"
+      two_player
+    else
+      puts "Enter 1-3"
+    end
+  end
+end
+
+def cpu
+  game  = Game.new(player_1 = Player::Computer.new("X"), player_2 = Player::Computer.new("O"), board = Board.new)
+  game.play 
+  if game.over?
+    puts "Does it really matter?!?!?!"
+  end
+  opening_statments
+end
+
+def one_player
+  puts "Want to go first? (y/n)"
+  input = gets.chomp
+  if input == "y"
+    game = Game.new(player_1 = Player::Human.new("X"), player_2 = Player::Computer.new("O"), board = Board.new)
+    game.play
+  elsif input == "n"
+    game = Game.new(player_1 = Player::Computer.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    game.play
+  else
+    "enter y for yes or n for no"
+  end 
+  opening_statments
+end
+
+def two_player
+  puts "Friends, get ready to duel"
+  game = Game.new(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+  game.play
+  opening_statments
+end
+
+play_game
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.